entries/t/ticketmaster.com.json

```
@@ -0,0 +1,12 @@
+{
+	"Ticketmaster": {
+		"domain": "ticketmaster.com",
+		"contact": {
+			"facebook": "Ticketmaster",
+			"twitter": "Ticketmaster"
+		},
+		"keywords": [
+			"entertainment"
```

Like with #6474, retail is probably the better category.
```suggestion
			"retail"
```